client/src/connection/rest/RestContentAdapter.ts

```diff
@@ -142,19 +150,27 @@ class RestContentAdapter implements ContentAdapter {
       ? []
       : await this.getChildItems(myFavoritesFolder);
 
-    const items = result.items.map(
-      (childItem: ContentItem, index): ContentItem => {
-        const favoriteUri = fetchFavoriteUri(childItem);
-        return {
-          ...childItem,
-          uid: `${parentItem.uid}/${index}`,
-          ...this.enrichWithDataProviderProperties(childItem, {
-            isInRecycleBin,
-            isInMyFavorites: parentIdIsFavoritesFolder || !!favoriteUri,
-            favoriteUri,
-          }),
-        };
-      },
+    const items = await Promise.all(
+      result.items.map(
+        async (childItem: ContentItem, index): Promise<ContentItem> => {
+          const favoriteUri = fetchFavoriteUri(childItem);
+          const enrichedItem = {
+            ...childItem,
+            uid: `${parentItem.uid}/${index}`,
+            ...this.enrichWithDataProviderProperties(childItem, {
+              isInRecycleBin,
+              isInMyFavorites: parentIdIsFavoritesFolder || !!favoriteUri,
+              favoriteUri,
+            }),
+          };
+
+          if (enrichedItem.contextValue?.includes("copyPath")) {
+            await this.updateUriWithFullPath(enrichedItem);
+          }
```

Comment:
It looks like the only time we _don't_ call `updateUriWithFullPath` is when...
 - getting root items
 - moving items

I'd advocate for moving this call inside `enrichWithDataProviderProperties` (I know that make things a _tiny_ bit more complicated since this function doesn't return a promise)